getting_started/introduction/introduction_to_godot.rst

```diff
@@ -0,0 +1,126 @@
+in this getting started series.
+
+What is Godot?
+--------------
+
+Godot is a general-purpose 2D and 3D game engine designed to support all sorts
+of projects. You can use it to create games or applications you can then release
+on desktop or mobile, as well as on the web.
+
+You can also create console games with it, although you either need strong
+programming skills or a developer to port the game for you.
+
+.. note:: The Godot team can't provide an open-source console export due to the
+          licensing terms imposed by console manufacturers. Regardless of the
+          engine you use, though, releasing games on consoles is always a lot of
+          work. You can read more on that here:
+
+What can the engine do?
+-----------------------
+
+Godot was initially developed in-house by an Argentinan game studio. Its
+development started in 2001, and the engine was rewritten and improved
+tremendously since its open-source release in 2014.
+
+Some examples of games created with Godot include Ex-Zodiac and Helms of Fury.
+
+.. image:: img/introduction_ex_zodiac.png
+
+.. image:: img/introduction_helms_of_fury.jpg
+
+As for applications, the open-source pixel art drawing program Pixelorama is
+powered by Godot, and so is the voxel RPG creator RPG in a box.
+
+.. image:: img/introduction_rpg_in_a_box.png
+
+You can find many more examples in the official showcase videos:
+
+- `April 2020 desktop and console showcase`_
+- `April 2020 mobile showcase`_
+
+How does it work and look?
+--------------------------
+
+Godot comes with a fully-fledged game editor with integrated tools to answer the
+most common needs. It includes a code editor, an animation editor, a tilemap
+editor, a shader editor, a debugger, a profiler, and more.
+
+.. image:: img/introduction_editor.png
+
+The team strives to offer a complete game creation package with a consistent
+user experience. While there is always room for improvement, the user interface
+keeps getting refined.
+
+Of course, if you prefer, you can work with external programs. We officially
+support importing 3D scenes designed in Blender_ and maintain plugins to code in
+VSCode_ and Emacs_ for GDScript and C#. We also support Visual Studio for C# on
+Windows.
+
+.. image:: img/introduction_vscode.png
+
+Programming languages
+---------------------
+
+Let's talk about the available programming languages.
+
+You can code your games using :ref:`GDScript <toc-learn-scripting-gdscript>`, a
+Godot-specific, a tightly integrated language with a lightweight syntax, or
+:ref:`C# <toc-learn-scripting-C#>`, which is popular in the games industry.
+These are the two main scripting languages we support.
+
+Godot also supports a node-based visual programming language named
+:ref:`VisualScript <toc-learn-scripting-visual_script>`.
+
+With the :ref:`GDNative <toc-tutorials-gdnative>` technology, you can also write
+gameplay or high-performance algorithms in C or C++ without recompiling the
```

Comment:
Should we mention in passing that this also allows for other languages supported by our community?